robot_skills/test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from ed_sensor_integration.srv import RayTrace
+from geometry_msgs.msg import PoseStamped
+from robot_skills import perception
+import rospy
+from std_msgs.msg import Header
+from tf import TransformListener
+
+rospy.init_node('dsada')
+
+p = perception.Perception("hero", None, None, camera_base_ns="hero/head_rgbd_sensor")
+
+srv_proxy = rospy.ServiceProxy("/hero/ed/ray_trace", RayTrace)
+
+tf_listener = TransformListener()
+
+while not rospy.is_shutdown():
```

Comment:
Unclear what is being tested in this file. No documentation, generic filename, etc. Also this looks like code for standalone usage, not a real test. (For in CI)